scripts/fetch_istio_releases.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+set -e
+
+RELEASE_DIR=${1}
+istio_version_array=(1.7.1 1.7.3 1.8.3 1.8.5 1.8.6 1.9.3 1.9.5 1.9.6 1.9.8 1.10.4)
+echo ${RELEASE_DIR}
+
+if [ -z "${RELEASE_DIR}" ]; then
+  echo "No directory given"
+  exit 1
+fi
+
+if [ -d "${RELEASE_DIR}" ] ; then
+  cd ${RELEASE_DIR}
+
+  dir=$(pwd)
```

Comment:
whats this do ?

Reply:
This is just for ease of use down the line - it just lets us pass in the directory we want to use to store the istio-releases. We can remove it its adding clutter - or are you specifically talking about dir=$(PWD)?

Reply:
you do mean dir - I see that is is unused in this file - I have removed it in my most recent commit